MantidQt/CustomInterfaces/inc/MantidQtCustomInterfaces/EnggDiffraction/EnggDiffractionViewQtGUI.h

```diff
@@ -154,6 +157,19 @@ class MANTIDQT_CUSTOMINTERFACES_DLL EnggDiffractionViewQtGUI
   void setDataVector(std::vector<boost::shared_ptr<QwtData>> &data,
                      bool focused) override;
 
+  void updateFittingDirVec(std::string &bankDir, std::string &focusedFile);
+
+  std::vector<std::string> splitFittingDirectory(Poco::Path selectedfPath);
```

Comment:
Ah, one good thing there would be to pass the Poco::Path as a const reference.
And yes, you can just use a forward declaration:
```
namespace Poco{
class Path;
}
```